python/lsst/ts/observatory/control/auxtel/atcalsys.py

```diff
@@ -0,0 +1,421 @@
+    ) -> None:
+        """Setup Monochromator.
+
+        Parameters
+        ----------
+        wavelength : `float`
+            Wavelength in nm.
+        entrance_slit : `float`
+            Size of entrance slit in mm.
+        exit_slit : `float`
+            Size of exist slit in mm.
+        grating : `int`
+            Grating to select.
+
+        """
+
+        await self.rem.atmonochromator.cmd_updateMonochromatorSetup.set_start(
+            wavelength=wavelength,
+            gratingType=grating,
+            fontExitSlitWidth=exit_slit,
+            fontEntranceSlitWidth=entrance_slit,
+            timeout=self.long_timeout,
+        )
+
+    async def electrometer_scan(self, duration: float) -> None:
+        """Perform an electrometer scan for the specified duration.
+
+        Parameters
+        ----------
+        duration : `float`
+            Total duration of scan.
+
+        Returns
+        -------
+        RunTimeError
+
+        """
+
+        self.rem.electrometer_1.evt_largeFileObjectAvailable.flush()
+
+        try:
+            await self.rem.electrometer_1.cmd_startScanDt.set_start(
+                scanDuration=duration, timeout=duration + self.cmd_timeout
+            )
+
+            # Make sure that a new lfo was created
+            await self.rem.electrometer_1.evt_largeFileObjectAvailable.next(
+                timeout=self.cmd_timeout, flush=False
+            )
+        except asyncio.TimeoutError:
+            raise RuntimeError("Electrometer scan issue.")
+
+    async def take_fiber_spectrum(
+        self,
+        delay: float,
+        integration_time: float,
+    ) -> None:
+        """Wait, then start an acquisition with the fiber spectrograph.
+
+        By default, this method will wait for `delay` seconds then start
+        an acquisition with the fiber spectrograph. Optionally the user may
+        provide a coroutine that will be awaited before the delay starts.
+
+        Parameters
+        ----------
+        delay : `float`
+            Seconds to wait before starting fiber spectrograph acquisition.
+        integration_time : `float`
+            Integration time for the fiber spectrum (seconds).
+
+        Returns
+        -------
+        RuntimeError : If exposure or return of lfo doesn't occur.
+        """
+        self.rem.fiberspectrograph_1.evt_largeFileObjectAvailable.flush()
+
+        await asyncio.sleep(delay)
+
+        try:
+            await self.rem.fiberspectrograph_1.cmd_expose.set_start(
+                integrationTime=integration_time,
+                timeout=integration_time + self.long_timeout,
+            )
+
+            self.rem.fiberspectrograph_1.evt_largeFileObjectAvailable.next(
+                timeout=self.long_timeout, flush=True
+            )
+        except asyncio.TimeoutError:
+            raise RuntimeError("Fiberspectrograph exposure issue.")
+
+    async def configure_camera(self, filter: str) -> None:
```

Comment:
I don't think you need this method here. You already have an instance of the camera so you should be good to set it internally.

Reply:
done as above